frontend/app/src/app/gallery/gallery.component.ts

```diff
@@ -12,4 +12,13 @@ export class GalleryComponent implements OnInit {
   ngOnInit(): void {
   }
 
+  changeOutput() {
+    if (document.getElementById("gallery").className == "square-gallery"){
+      document.getElementById("gallery").className = "list-gallery";
```

Comment:
Why are you directly manipulating the DOM instead of using the component you defined above? I think that this somehow defeats the philosophy behind angular and might lead to inconsistencies (you might be doing things behind the "component's back").  But I am not an expert in Angular, feel free to confirm that this is the best practice.